assets/base/i18n/en.json

```diff
@@ -183,7 +183,6 @@
   "mobile.channel_list.not_member": "NOT A MEMBER",
   "mobile.channel_list.unreads": "UNREADS",
   "mobile.channel_members.add_members_alert": "You must select at least one member to add to the channel.",
-  "mobile.channel.markAsRead": "Mark As Read",
```

Comment:
????

Reply:
Only place this was being used was the channel peek preview popup.